test/unit/app/app.test.js

```diff
@@ -596,6 +603,14 @@ describe('App', () => {
       it('should return null for permsOfLoggedInUser', () => {
         expect(result.permsOfLoggedInUser).to.be.null;
       });
+
+      it('should map working.resendingEmailVerification.inProgress to resendingEmailVerification', () => {
```

Comment:
Looks like you used `resendingEmailVerification` as the expected prop here.  This should be renamed to match the prop, or vice-versa.

Reply:
Matching the new renamed prop above